pkg/event/reader/reader.go

```diff
@@ -24,10 +22,6 @@ func NewEventReader(src interface{}) EventReader {
 	switch src := src.(type) {
 	case *kernel.Probe:
 		return NewKernelReader(src)
-	case *redis.PubSub:
-		return NewRedisReader(src)
-	case *topology.Topology:
-		return NewTopologyReader(src)
```

Comment:
A lot of this is legacy from a while back - I'm probably going to remove this entire API as it is redundant.